realm/realm-library/src/main/java/io/realm/log/RealmLog.java

```diff
@@ -29,12 +27,8 @@
 @Keep
 public final class RealmLog {
 
-    private static final Logger[] NO_LOGGERS = new Logger[0];
-
-    // All of the below should be modified together under under a lock on LOGGERS.
-    private static final List<Logger> LOGGERS = new ArrayList<>();
-    private static volatile Logger[] loggersAsArray = NO_LOGGERS;
-    private static int minimumNativeLogLevel = Integer.MAX_VALUE;
+    @SuppressWarnings("FieldCanBeLocal")
```

Comment:
This is not. just to suppress the lint warning that AS asks to move it to a local var.